### DISCUSSION TITLE
Filter list question

### DISCUSSION THREAD

**Discussion by TheCeet** [AdguardTeam/AdGuardHome#4075] [Q&A] [answered]
When you have per example 10 blocklists. Does AdGuard Home look for dubble entries and make 1 giant 'database/file' with domains or not?
Pi-Hole filters all unique domains to 1 list with unique domains.

I searched for an answer on reddit or the wiki, but I couldn't find an answer.

**Comment by ainar-g** [comment, marked as answer]
See #3074.  Duplicates are removed from the search index.
